### DISCUSSION TITLE
Can buildpack used for ai model which is developed by PyTorch?

### DISCUSSION THREAD

**Discussion by Yan0613** [buildpacks/community#262] [General]
Can buildpack used for ai model which is developed by PyTorch?

**Comment by natalieparellano** [comment]
I know some folks are using buildpacks for AI things, but I am not sure about PyTorch. I'm not aware of a PyTorch buildpack - you would probably need to make your own. A brief introduction can be found here: https://buildpacks.io/docs/for-buildpack-authors/ or in the #buildpacks-authors channel on https://slack.cncf.io/.
